www/index.rst

```diff
@@ -184,14 +184,26 @@ particular, these are some of the core packages:
   </div>
   </li>
 
+  <li class="span12">
+  <div class="thumbnail">
+  <div class="pull-left img">
+    <a href="http://pandas.pydata.org/">
```

Comment:
Ralf, this is a URL for Pandas.  Shouldn't this link to NumFOCUS?

Reply:
oops. thanks for catching that. fixed in gh-293